src/App.tsx

```diff
@@ -722,18 +722,15 @@ function App(): ReactElement {
                 disabled={disableTimeControlsUi}
                 wrapIncrement={true}
               />
-              <div style={{ display: "flex", flexDirection: "row", alignItems: "center" }}>
-                <span className={styles.verticalDivider} style={{ height: "24px", margin: "0 8px" }}></span>
-                <div style={{ width: "200px", maxWidth: "60vw" }}>
-                  <PlaybackSpeedControl
-                    fps={playbackFps}
-                    onChange={(fps) => {
-                      setPlaybackFps(fps);
-                      timeControls.setPlaybackFps(fps);
-                    }}
-                    disabled={disableTimeControlsUi}
-                  />
-                </div>
+              <div style={{ display: "flex", flexDirection: "row", flexGrow: 1, justifyContent: "flex-end" }}>
+                <PlaybackSpeedControl
```

Comment:
I always find it disappointing when you can't just drop in a component without N different wrapper elements.  I suppose it's all about careful positioning.

Reply:
This bit just aligns it with the left hand corner of the row for nice spacing-- it's totally optional but I found it looked better than just having it be part of the row.

At some point soon I should just make reusable div components for all of the repeated `flex` boilerplate I have 😋